build.gradle

```diff
@@ -98,6 +98,18 @@ configure(javaProjects) {
             dependency("org.opensaml:opensaml:2.6.4")
             dependency("org.python:jython-standalone:2.7.1")
             dependency("org.springframework.security.extensions:spring-security-saml2-core:1.0.3.RELEASE")
+            dependencySet(group: "com.fasterxml.jackson.core", version: "2.9.5") {
```

Comment:
We can discuss why you're upgrading Jackson from 2.8.x to 2.9.x in person but the easier way to do this is to set `jackson.version` in `gradle.properties` similar to how you did the `h2.version`

See the boot bom the property comes from: https://github.com/spring-projects/spring-boot/blob/v1.5.13.RELEASE/spring-boot-dependencies/pom.xml#L99